stdlib/3.4/asyncio/subprocess.pyi

```diff
@@ -46,9 +46,9 @@ class Process:
 @coroutine
 def create_subprocess_shell(
     *Args: Union[str, bytes],  # Union used instead of AnyStr due to mypy issue  #1236
-    stdin: int = ...,
-    stdout: int = ...,
-    stderr: int = ...,
+    stdin: Union[int, IO[Any]] = ...,
```

Comment:
None should also be allowed (it's the default value for each of these parameters).